iocore/hostdb/I_HostDBProcessor.h

```diff
@@ -361,7 +361,13 @@ struct HostDBRoundRobin {
   unsigned short current;
   ink_time_t timed_rr_ctime;
 
-  HostDBInfo info[];
+  // This is the equivalent of a variable length array, we can't use a VLA because
+  // HostDBInfo is a non-POD type-- so this is the best we can do.
+  HostDBInfo *
+  info()
```

Comment:
Looking at the usage pattern, I think this would be better:

```C
HostDBInfo * info(unsigned n) { ... }
```

This lets you avoid the tedious ``info()[n]`` notation, and also lets you check the offset at runtime if necessary (at least a debug assertion should be here).